### ISSUE TITLE
Conan Package for Release 0.62

### ISSUE THREAD

**Issue by LvdStokker** [rbock/sqlpp11#495]
Hi @rbock,

Are you perhaps creating a new Conan package for your lastest 0.62 release or are aware of anyone who is?

**Comment by rbock** [comment]
Hi Luke,

I am not involved in Conan packaging. Maybe the package itself contains data about the maintainer?

Best,
Roland

**Comment by LvdStokker** [comment]
Hi Roland,

No problem. I believe I was able to track down the initial contributor to the package and I have contacted him for assistance. Alternatively, I may create the latest package myself, or work with him to create it.

I am actually a colleague of @CJCombrink who has made some recent contributions to your library and so we would likely want to see these changes soon in a new 0.63 package for our users as well. So taking over Conan package creation may be something we end up doing together in any case as we build dependencies on your library.

Thanks for a great library btw!
